installer/datafiles/Linux.data

```diff
@@ -58,9 +66,14 @@ fi
 #if DISABLE_PORT != true
```

Comment:
Do we support disabling the port anymore? I think this was for trying to support Azure, but we ended up coming up with the more flexible scheme we have now (listen or don't listen, configurable, no listen by default).

Reply:
I see support code for that, but I don't think we ever actually invoke it via the configure script. Please validate and, if so, let's get rid of this. It's just confusing.